spec/motions-spec.coffee

```diff
@@ -1744,6 +1745,38 @@ describe "Motions", ->
       keydown(',')
       expect(editor.getCursorScreenPosition()).toEqual [0, 2]
 
+    it "gets state from GlobalVimState", ->
```

Comment:
yup, done too. All good catches, thank you. I also changed a find to a till to test both.